### DISCUSSION TITLE
一点点对多博客程序支持的问题

### DISCUSSION THREAD

**Discussion by sysfox** [Qexo/Qexo#308] [Ideas]
刚看到 @am-abudu 的提交 #f27a5d5eac91907b3e4aa8da8d95f8017d2d9ac0
有两个小小的问题：

1.后面会支持Hugo吗？
2.或者，支持同时管理两个Hexo；一个Hexo一个Hugo；两个Hugo？（更大的工程（逃

**Comment by am-abudu** [comment]
这项功能还在测试中，请你可以为 [这个文件](https://github.com/Qexo/Qexo/blob/dev/hexoweb/libs/platforms/configs.py) 提交Pr

**Reply by sysfox** [reply]
@am-abudu 已提交PR #309 ，不确保完全可用

**Comment by sysfox** [comment]
说句闲话，你这都支持多博客程序了，那Qexo要改名了吧(doge

**Comment by sysfox** [comment]
所以以后会不会添加那么亿点点代码支持同时管理多个程序？

**Reply by am-abudu** [reply]
建议部署两个

**Comment by sysfox** [comment]
让我想想还有啥子博客程序
Vuepress: https://github.com/vuepress/vuepress-next/
Jeklly: https://jekyllcn.com/docs/home/
Vitepress
